### PR TITLE
Added correct stm32 board config. It's f4 instead of f3

### PR DESCRIPTION
**Pull request by JGoard**
Sort of self-explanatory. Issues with gdb and openocd still need to be resolved with regards to how the program seems to be lost when debugging. Breakpoints are set, but are never reached however.

Perhaps some compiler flags  in the makefile could be changed for better debugging symbols? I am not sure at this time.

Keep in mind, the chip still is an ARM Cortex-M4. The only thing that has changed is the stm32 board.cfg in the launch json.

*TL;DR* : use "board/st_nucleo_f3.cfg" for openocd purposes.

image

### CODE REVIEW

**Review by nick-a-schneider — APPROVED** [review]
Looks good to me! I can't verify it actually works until my hardware get here, but assuming it works for you lets merge.
